cpp/basix/finite-element.h

```diff
@@ -725,24 +754,30 @@ class FiniteElement
            std::vector<std::tuple<std::vector<std::size_t>, std::vector<double>,
                                   xt::xtensor<double, 2>>>>
       _etrans_invT;
+
+  /// Indicates whether or not this is the discontinuous version of the element
```

Comment:
```suggestion
  // Indicates whether or not this is the discontinuous version of the element
```
Double forward slash only for private variables.